docs/Contribute.md

```diff
@@ -0,0 +1,126 @@
+
+### Styling
+
+**Links:**
+
+* Absolute: `[link text](www.destinationURL.com)`  
+_Example:_ `[Broad Institute](www.broadinstitute.org)` _produces this link_ [Broad Institute](www.broadinstitute.org).
+* Relative: `[link text](Destination_Page)`, where `Destination_Page` is the file name without the `.md` extension  
+_Example:_ `[How to use the Cromwell CLI](CommandLine)` _produces this link_ [How to use the Cromwell CLI](CommandLine).
+* Anchor link: `[anchor text](../Path/To/Page#Anchor)`  
+_Example:_ `[HPC filesystems](backends/HPC#filesystems)` _produces this link_ [HPC filesystems](backends/HPC#filesystems).
+
+**Code:**
+
+* To style a word of code, use a backtick (\`) before and after the word.  
+_Example:_ \`file.json\` _produces_ `file.json`. 
+* To style a block of code, use three backticks (\`\`\`) before and after the block of code.  
+_Example:_  
+	\`\`\`  
+		workflow myWorkflow {  
+		call myTask  
+		}  
+	\`\`\`  
+_produces this block_  
+```  
+	workflow myWorkflow {  
+	call myTask  
+	}  
+```  
+* To use syntax highlighting, include the language after the first three backticks (\`\`\`).  
+_Example:_   
+	\`\`\`json  
+		\{  
+		    "MyWorkflow.MyTask.VariableTwo": "Variable2"  
+		\}  
+	\`\`\`	
+_produces this block_  
+```json  
+	{
+	    "MyWorkflow.MyTask.VariableTwo": "Variable2"
+	}
+```	
+
+**REST API:**
+
+1. Edit the [`cromwell.yaml`](https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/resources/swagger/cromwell.yaml) to make any changes to the REST API content.  
+2. Regenerate the REST API markdown file by running `sbt generateRestApiDocs` from the main Cromwell directory.
+3. Commit both the changes to the `cromwell.yaml` and the (re)generated [RESTAPI.md](https://github.com/broadinstitute/cromwell/blob/develop/docs/api/RESTAPI.md).
+4. Once your branch is merged to the [`develop` branch](https://github.com/broadinstitute/cromwell/tree/develop), you will see your changes on the [REST API page](api/RESTAPI/).
+
+**Left-side menu:**
+
+To add or remove items from the menu, edit [mkdocs.yml](https://github.com/broadinstitute/cromwell/blob/develop/mkdocs.yml) in Cromwell.
+
+**Images**
+
+* Relative: `![](ImgName.png)`
+	* _Example:_ `![](jamie_the_cromwell_pig.png)` _produces this image_  
+	![](jamie_the_cromwell_pig.png)  
+* Absolute: `![](URLofImg.png)`
+	* _Example:_ `![](https://www.broadinstitute.org/sites/all/themes/custom/at_broad/logo.png)` _produces this image_  
+	![](https://www.broadinstitute.org/sites/all/themes/custom/at_broad/logo.png)
+
+### FAQs
+
+**_Why isn't my documentation showing up?_**
+
+* **Have you confirmed that your PR is merged?**  
+If not, [go merge it](https://github.com/broadinstitute/cromwell/pulls). This will trigger an automatic build.  
```

Comment:
Only certain team members can merge. Others should wait patiently for a merge and then yell if we take too long to notice their stale PR.

Reply:
I thought everyone could merge once it had been approved 2x?

Reply:
In particular, external contributions **can not** merge themselves. Since presumably they're the crowd who would care about this sort of information, that's a handy difference to note